surveys/contextual-survey-metadata.json

```diff
@@ -35,5 +35,42 @@
                 "promptRemainsVisible": true
             }
         ]
+    },
+    {
+        "uniqueId": "new-inspector-survey-q2-2025",
+        "startDate": "2025-03-05T09:00:00-07:00",
+        "endDate": "2025-04-01T09:00:00-07:00",
+        "description": "Share your feedback on the new Flutter Inspector with a 5-question survey.",
+        "snoozeForMinutes": 1440,
```

Comment:
1440 minutes = 24 hours

Reply:
what does snooze do? delay showing the user the survey if they have seen the notification but never interacted with it?